app/src/main/java/com/owncloud/android/operations/CreateFolderOperation.java

```diff
@@ -292,6 +307,20 @@ private RemoteOperationResult normalCreate(OwnCloudClient client) {
 
             createdRemoteFolder = (RemoteFile) remoteFolderOperationResult.getData().get(0);
             saveFolderInDB();
+
+            OCFile folder = getStorageManager().getFileByDecryptedRemotePath(remotePath);
+
+            if (encrypted) {
+                RemoteOperationResult remoteOperationResult = new ToggleEncryptionRemoteOperation(folder.getLocalId(),
```

Comment:
```suggestion
            if (encrypted) {
                final OCFile folder = getStorageManager().getFileByDecryptedRemotePath(remotePath);
                final RemoteOperationResult remoteOperationResult = new ToggleEncryptionRemoteOperation(folder.getLocalId(),
```

No need to call the storage manager if encrypted is false